pkg/tenant/api/v1/api.go

```diff
@@ -66,6 +66,14 @@ type TenantClient interface {
 // Top Level Requests and Responses
 //===========================================================================
 
+// Confirmation allows APIs to protect users from unintended actions such as deleting
+// data by including a confirmation token in the request.
+type Confirmation struct {
+	ID           string `json:"id"`
+	Name         string `json:"name"`
```

Comment:
Is the name something you're thinking the user should supply?

Reply:
I think it would be part of the response, so the user shouldn't supply it in the request but it gives the frontend something to display in the confirmation prompt without having to reference the original object.